### PR TITLE
fix, unsupported strings in alertmanager keys

### PR DESCRIPTION
**Pull request by epcim**
ref: #77

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area config

> /area outputs

> /area tests

> /area helm

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by epcim** [comment]
what is contrib policy, how to sign-off ? Details link fails.. (will fix over weekend )

File: outputs/alertmanager.go [hunk 1]
```diff
@@ -24,8 +24,9 @@ func newAlertmanagerPayload(falcopayload types.FalcoPayload) []alertmanagerPaylo
 	for i, j := range falcopayload.OutputFields {
 		switch j.(type) {
 		case string:
-			//AlertManger doesn't support dots in a label name
-			amPayload.Labels[strings.Replace(i, ".", "_", -1)] = j.(string)
+			//AlertManger unsupported chars in a label name
+			replacer := strings.NewReplacer(".", "_", "[", "", "]", "")
```
Comment (Issif) [inline outputs/alertmanager.go:28]:
```suggestion
			replacer := strings.NewReplacer(".", "_", "[", "_", "]", "")
```
I think it's better to replace `proc_aname[2]` by `proc_aname_2`, what do you think?

**Review by Issif — CHANGES_REQUESTED** [review]

**Comment by Issif** [comment]
> what is contrib policy, how to sign-off ? Details link fails.. (will fix over weekend )

The easiest is :

```
git commit --amend --signoff
git push --force
```

File: outputs/alertmanager.go (hunk 1 above)
Comment (epcim) [reply outputs/alertmanager.go:28]: updated

**Review by Issif — APPROVED** [review]

**Pull request merged**